tools/packaging/static-build/qemu/build-base-qemu.sh

```diff
@@ -9,6 +9,8 @@ set -o nounset
 set -o pipefail
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly repo_root_dir="$(cd "${script_dir}/../../../.." && pwd)"
+readonly qemu_builder="${script_dir}/build-qemu.sh"
```

Comment:
Do we need the `readonly qemu_builder="${script_dir}/build-qemu.sh"` if we are running the whole file path in the 'docker' run to build it?

Reply:
Not really, good catch.